eng/targetingpacks.targets

```diff
@@ -100,21 +104,13 @@
   </Target>
 
   <!--
-    SDK tries to download runtime packs when RuntimeIdentifier is set, remove them from PackageDownload item.
-    TODO: Remove this target when an 8.0.100 SDK is consumed that respects EnableRuntimePackDownload.
+    SDK tries to download the ILCompiler runtime packs.
+    TODO: Remove this target when a 9.0.100 SDK is consumed that respects EnableRuntimePackDownload for the ILCompiler.
    -->
   <Target Name="RemoveRuntimePackFromDownloadItem"
-          Condition="'$(UseLocalTargetingRuntimePack)' == 'true'"
+          Condition="'$(UseLocalILCompilerPack)' == 'true'"
           AfterTargets="ProcessFrameworkReferences">
     <ItemGroup>
-      <PackageDownload Remove="@(PackageDownload)"
-                       Condition="'$(UsePackageDownload)' == 'true' and $([System.String]::Copy('%(Identity)').StartsWith('$(LocalFrameworkOverrideName).Runtime'))" />
-      <PackageReference Remove="@(PackageReference)"
-                        Condition="'$(UsePackageDownload)' != 'true' and $([System.String]::Copy('%(Identity)').StartsWith('$(LocalFrameworkOverrideName).Runtime'))" />
-      <PackageDownload Remove="@(PackageDownload)"
-                       Condition="'$(UsePackageDownload)' == 'true' and $([System.String]::Copy('%(Identity)').StartsWith('$(LocalFrameworkOverrideName).Crossgen2'))" />
-      <PackageReference Remove="@(PackageReference)"
-                        Condition="'$(UsePackageDownload)' != 'true' and $([System.String]::Copy('%(Identity)').StartsWith('$(LocalFrameworkOverrideName).Crossgen2'))" />
       <PackageDownload Remove="@(PackageDownload)"
                        Condition="'$(UsePackageDownload)' == 'true' and $([System.String]::Copy('%(Identity)').EndsWith('Microsoft.DotNet.ILCompiler'))" />
```

Comment:
We can remove this PackageDownload and change the PackageReference removal below to be:

```
<PackageReference Remove="Microsoft.DotNet.ILCompiler" Condition="'%(PackageReference.IsImplicitlyDefined)' == 'true'" />
```